addons/l10n_ua/data/template/account.tax-ua_psbo.csv

```diff
@@ -1,79 +1,79 @@
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat14_psbo","20","Придбання ПДВ 14%","","+ ПДВ 14%","14.0","purchase","tax_group_vat14","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat14_psbo","20","ПДВ 14%","","+ ПДВ 14%","14.0","purchase","tax_group_vat14","","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6442"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat14incl_psbo","20","Придбання в т. ч. ПДВ 14%","","в т. ч. ПДВ 14%","14.0","purchase","tax_group_vat14","True","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat14incl_psbo","20","в т. ч. ПДВ 14%","","в т. ч. ПДВ 14%","14.0","purchase","tax_group_vat14","True","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6442"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat7_psbo","21","Придбання ПДВ 7%","","+ ПДВ 7%","7.0","purchase","tax_group_vat7","","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat7_psbo","21","ПДВ 7%","","+ ПДВ 7%","7.0","purchase","tax_group_vat7","","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6442"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat7incl_psbo","21","Придбання в т. ч. ПДВ 7%","","в т. ч. ПДВ 7%","7.0","purchase","tax_group_vat7","True","base","invoice",""
-"","","","","","","","","","tax","invoice","ua_psbp_6441"
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat7incl_psbo","21","в т. ч. ПДВ 7%","","в т. ч. ПДВ 7%","7.0","purchase","tax_group_vat7","True","base","invoice",""
+"","","","","","","","","","tax","invoice","ua_psbp_6442"
 "","","","","","","","","","base","refund",""
-"","","","","","","","","","tax","refund","ua_psbp_6441"
-"purchase_tax_template_vat0_psbo","22","Придбання ПДВ 0%","","ПДВ 0%","0.0","purchase","","","base","invoice",""
+"","","","","","","","","","tax","refund","ua_psbp_6442"
+"purchase_tax_template_vat0_psbo","22","ПДВ 0%","","ПДВ 0%","0.0","purchase","","","base","invoice",""
 "","","","","","","","","","tax","invoice",""
 "","","","","","","","","","base","refund",""
 "","","","","","","","","","tax","refund",""
-"purchase_tax_template_vat_free_psbo","23","Придбання звільнене від  ПДВ","","Звільнено від ПДВ","0.0","purchase","","","base","invoice",""
+"purchase_tax_template_vat_free_psbo","23","Звільнено від  ПДВ","","Звільнено від ПДВ","0.0","purchase","","","base","invoice",""

```

Comment:
We try to keep the Tax names as short as can be. 
Even if it means completing the tax description field to explain them if necessary.

Format = Rate% (Territory) (Scope) (Custom)
Territory (fiscal position) : 
Empty for domestic
EU for intra-community
EX for extracom/export/import
Scope
M for merchandise
G for goods
S for services
IG for investment goods
IS for investment services
Custom: if needed
Various specific abbreviation conventions:
Withholding => WH
Withholding income tax => WHI
Included/TTC => INC
Dividend distribution => DD
Triangular sale => Tri
Reverse charge => RC
Real Estate => RE

Examples
Sales Taxes :
21% => 21%
6% S. => 6% S
0% Cocont. => 0% Cocont

Purchase Taxes :
6% S. => 6% S
0% Cocont. S. => 0% S.Cocont
21% M. => 21% M
21% Non EU - Bien d'investissement => 21% EX IG
12% EU - Bien divers => 12% EU G
50% Non Déductible - Frais de voiture => 21% Car
10% Withholding tax royalties => 10% WH Royalties